### PR TITLE
frontend: sidebar reorder

### LINKED ISSUE #1613: frontend: sidebar order workflow improvements
Sidebar Storage:
Change the order from StorageClass,PV,PVC to PVC,PV,SC

Sidebar Configuration:
image

What does anyone think?

### PR DESCRIPTION
**Pull request by farodin91**
Fixes #1613 

Sidebar
image
image
image

### CODE REVIEW

**Comment by farodin91** [comment]
I will split into multiple PRs

**Comment by farodin91** [comment]
See #1620 and #1619

File: frontend/src/i18n/locales/de/glossary.json [hunk 1]
```diff
@@ -181,14 +181,13 @@
   "Access Modes": "Zugriffsmodi",
   "Volume Mode": "Speichervolume-Modus",
   "Storage Class": "Speicher-Klasse",
-  "Volume Claims": "Volume Claims",
+  "Volume Claims": "",
```
Comment (joaquimrocha) [inline frontend/src/i18n/locales/de/glossary.json:184]: I think these were mistakenly rewritten as empty (same for all translation languages).

**Review by joaquimrocha — CHANGES_REQUESTED** [review]

**Comment by farodin91** [comment]
@joaquimrocha I have updated the translation and reorder a bit workloads.

**Review by joaquimrocha — APPROVED** [review]

**Pull request merged**